jsapp/xlform/src/view.params.templates.coffee

```diff
@@ -1,9 +1,12 @@
 module.exports = do ->
-  numberParam = (label, number) ->
+  numberParam = (label, number, defaultValue) ->
+    if typeof defaultValue isnt 'undefined'
+      defaultValueAttr = "placeholder='#{defaultValue}'"
```

Comment:
Could we make the default value actually a value instead of a placeholder? Right now, for example, if I change `start` and `step` but leave `end` at its default of `10`, I end up with `"parameters": "start=-2;step=3"` in the content and `start=-5;step=3` in the XLS. I don't think this is intuitive, and who knows if some future client or XLSForm converter won't follow the spec (or the spec will change :smiling_imp:)

Reply:
I'm using default values when parameters are empty and the placeholder is still there, so it is more clear to user that after removing the input value and saving the form, it will still be `10`. It would probably be easies if you check how it feels live :)